standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/conf/TestMetastoreConf.java

```diff
@@ -482,6 +483,8 @@ public void dumpConfig() throws IOException {
   public void testClassNames() {
     Assert.assertEquals(MetastoreConf.DEFAULT_STORAGE_SCHEMA_READER_CLASS,
         DefaultStorageSchemaReader.class.getName());
+    Assert.assertEquals(MetastoreConf.SERDE_STORAGE_SCHEMA_READER_CLASS,
```

Comment:
Can you add a unit test/q test with the failed test case? Also, it would be nice to add other storage format options along with Avro.